processed 5 tasks

init:
A: object(103)

task 1 'publish'. lines 8-23:
created: object(105), object(106), object(107)
mutated: object(104)

task 2 'programmable'. lines 25-27:
created: object(109)
mutated: object(107), object(108)

task 3 'programmable'. lines 29-31:
Error: Transaction Effects Status: Move Runtime Abort. Location: sui::balance::increase_supply (function index 3) at offset 12, Abort Code: 1
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: MoveAbort(MoveLocation { module: ModuleId { address: sui, name: Identifier("balance") }, function: 3, instruction: 12, function_name: Some("increase_supply") }, 1), source: Some(VMError { major_status: ABORTED, sub_status: Some(1), message: Some("sui::balance::increase_supply at offset 12"), exec_state: Some(ExecutionState { stack_trace: [(Some(ModuleId { address: sui, name: Identifier("coin") }), FunctionDefinitionIndex(17), 5)] }), location: Module(ModuleId { address: sui, name: Identifier("balance") }), indices: [], offsets: [(FunctionDefinitionIndex(3), 12)] }), command: Some(0) } }

task 4 'programmable'. lines 33-34:
Error: Transaction Effects Status: Invalid command argument at 1. Invalid usage of value. Mutably borrowed values require unique usage. Immutably borrowed values cannot be taken or borrowed mutably. Taken values cannot be used again.
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 1, kind: InvalidValueUsage }, source: None, command: Some(0) } }
